health/warnings.go

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package health
+
+import (
+	"fmt"
+)
+
+/**
+This file contains definitions for the Warnables maintained within this `health` package.
+*/
+
+// updateAvailableWarnable is a Warnable that warns the user that an update is available.
+var updateAvailableWarnable = Register(&Warnable{
+	Code:     "update-available",
+	Title:    "Update available",
+	Severity: SeverityMedium,
```

Comment:
all these severities seem kinda random. Like maybe this is Low unless it's declared as a security update? maybe we need SeverityFunc that when non-nil takes precedence over the static Severity field and can be dynamic based on the args. TODO perhaps.

Reply:
Changed this to Low, and added a TODO comment to implement a SeverityFunc